tsconfig.spec.json

```diff
@@ -1,11 +1,11 @@
 {
   "extends": "./tsconfig.json",
   "include": ["./jest/testing-library__jest-dom.d.ts", "./jest/verifyComponent.d.ts", "**/*.spec.tsx", "**/*.spec.ts"],
-  "exclude": [],
+  "exclude": ["cypress"],
```

Comment:
Cypress has an explicit type check command. Leaving this out means that Cypress spec files are checked without defining the Cypress type interface which errors on `cy is not defined`.